lib/providers/services/http_service.dart

```diff
@@ -245,8 +299,6 @@ class HttpService {
 
     final String proposalsURL = '$baseURL/v1/chain/get_table_rows';
 
-    // final String minimumStake = "1.0000 SEEDS";
-
```

Comment:
Checking stake would be smart. Can we do that?

Reply:
I am not sure if we need that - we already have different stages and states of proposals - and it's not related to harvest pull request anyway